docker-compose.yml

```diff
@@ -2,7 +2,7 @@ version: "3.8"
 
 services:
   conda-store-worker:
-    image: quansight/conda-store-server:latest
+    image: quansight/conda-store-server:${CONDA_STORE_SERVER_VERSION:-2023.10.1}
```

Comment:
This means every time we make a release, we also need to manually update the version used here, correct?